modules/integration_azure-cdn/conf/01-origin-health.yaml

```
@@ -0,0 +1,24 @@
+module: Azure CDN
+name: origin health
+filtering: "filter('resource_type', 'Microsoft.Cdn/profiles') and filter('primary_aggregation_type', 'true')"
+id: cdn_origin_health
+value_unit: "%"
+
+transformation: true
+aggregation: ".mean(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
```

Otherwise, having a backend unhealthy one minute and healthy again will trigger the detector.
```suggestion
aggregation: ".max(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
```

---

Also (case to double check - taken from MS documentation)
```suggestion
aggregation: ".mean(by=['Origin', 'azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
```